<launch>
    <!-- launch specification file to get parameters for the task -->
    <include file="$(find uniform_object_rearrangement)/launch/task_specification.launch"></include>

    <!-- run an example -->
    <!-- arg (1)#object (2)instance_id (3)generate a new('g') or load an existing('l') instance 
            (4)method_name -->
    <node pkg="uniform_object_rearrangement" type="MonotoneTest.py" name="monotone_test" 
        args="$(arg monotone_test)" output="screen" required="True" />

    <node pkg="uniform_object_rearrangement" type="PybulletExecutionScene.py" name="pybullet_execution_scene"
		    output="screen" required="True" />

    <node pkg="uniform_object_rearrangement" type="PybulletPlanScene.py" name="pybullet_plan_scene"
		    output="screen" required="True" />

    <node pkg="uniform_object_rearrangement" type="PoseEstimator.py" name="fake_pose_estimator"
        output="screen" required="False" />

    <node pkg="uniform_object_rearrangement" type="main_planner_node" name="main_planner" output="screen" />
   
</launch>